src/Users/Views/user_settings.php

```diff
@@ -34,8 +34,8 @@
                     <div class="col-12 col-sm-4">
                         <div class="form-check">
                             <input class="form-check-input" type="checkbox" name="emailActivation"
-                                   value='Shield\Authentication\Actions\EmailActivate' id="email-activation"
-                                <?php if (old('emailActivation', setting('Auth.actions')['register']) === 'Shield\Authentication\Actions\EmailActivate') : ?>
+                                   value='Bonfire\Auth\Authentication\Actions\EmailActivator' id="email-activation"
+                                <?php if (old('emailActivation', setting('Auth.actions')['register']) === 'Bonfire\Auth\Authentication\Actions\EmailActivator') : ?>
```

Comment:
As per comments above, I'm not sure this is really needed and would require a maintenance headache.

Reply:
this is related to Themeable trait end setting templates